src/main/webapp/resources/js/components/samples/components/SampleFileContenate.tsx

```diff
@@ -131,8 +104,8 @@ export function SampleFileConcatenate({
       }
     );
 
-    if (pairedCount === numFilesSelected) return true;
-    if (singleEndCount === numFilesSelected) return true;
+    if (pairedCount === numFilesSelected || singleEndCount === numFilesSelected)
+      return true;
```

Comment:
NIT: This is ugly, and hard to read, it should be shortened to:

```javascript
return pairedCount === numFilesSelected || singleEndCount === numFilesSelected;
```

Reply:
Very ugly indeed. Updated in 152ba864e8160e1aec8276f84e433db80e1d2f1b